google-cloud-firestore/lib/google/cloud/firestore/client.rb

```diff
@@ -649,18 +649,18 @@ def transaction max_retries: nil, commit_response: nil
             transaction = Transaction.from_client \
               self, previous_transaction: transaction.transaction_id
             retry
-          rescue Google::Cloud::InvalidArgumentError => err
+          rescue Google::Cloud::InvalidArgumentError => e
             # Return if a previous call was retried but ultimately succeeded
-            return nil if backoff[:current] > 0
+            return nil if (backoff[:current]).positive?
```

Comment:
Why are the parens needed?

Reply:
Added by autocorrect, no idea. I'll remove.

Reply:
Fixed, thanks!